charts/garnet/values.yaml

```diff
@@ -0,0 +1,122 @@
+# Default values for garnet.
+
+statefulSet:
+  # -- StatefulSet annotations
+  annotations: {}
+  # -- StatefulSet replicas
+  replicas: 1
+  # -- StatefulSet revisionHistoryLimit
+  revisionHistoryLimit: 1
+  updateStrategy:
+    # -- StatefulSet updateStrategy type
+    type: RollingUpdate
+
+image:
+  # -- Image registry
+  registry: ghcr.io
+  # -- Image repository
+  repository: microsoft/garnet
+  # -- Image pull policy
+  pullPolicy: IfNotPresent
+  # -- Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+# -- Image pull secrets
+imagePullSecrets: []
+# -- Chart name override
+nameOverride: ""
+# -- Chart full name override
+fullnameOverride: ""
+
+serviceAccount:
+  # -- Specifies whether a service account should be created
+  create: false
+  # -- Annotations to add to the service account
+  annotations: {}
+  # -- The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+  # -- Creates the token object
+  token: false
+
+containers:
+  # -- Containers args
+  args: ["--port", "6379", "-m", "128m", "-i", "128m", "--auth", "Password","--password", "xxxxxxxxxx"]
```

Comment:
nit: does it make sense to omit this for default? First time consumers might find it easier to just install and use this rather than seeing why they might need a password?

Reply:
Additionally, sensitive data like password should be stored in a secret object instead of being passed as arguments.

Reply:
I am closing this since it seems to have been resolved by the owner of the PR